examples/dpo_humanlike/dpo.yaml

```
@@ -1,17 +1,19 @@
+project: "dpo_example"
+name: "trinity_dpo"
 mode: train
-global_config:
-  total_epochs: 20
-  batch_size: 32 # NOTE
+algorithm:
   algorithm_type: dpo
+checkpoint_root_dir: /PATH/TO/CHECKPOINT
 model:
-  model_path: '/PATH/TO/MODEL/CHECKPOINT/' # NOTE
+  model_path: '/PATH/TO/MODEL' # NOTE
```

为了一致，yaml里的 # NOTE可以都删掉，没啥用
```suggestion
  model_path: '/PATH/TO/MODEL' 
```